README.md

```diff
@@ -1,86 +1,68 @@
+        replyId: event.message.id,
+      ));
     }
   });
 }
 ```
 
 ## Other nyxx packages
 
-- [nyxx_interactions](https://github.com/nyxx-discord/nyxx_interactions)
-- [nyxx_commander](https://github.com/nyxx-discord/nyxx_commander)
-- [nyxx_extensions](https://github.com/nyxx-discord/nyxx_extensions)
-- [nyxx_lavalink](https://github.com/nyxx-discord/nyxx_lavalink)
-- [nyxx_pagination](https://github.com/nyxx-discord/nyxx_pagination)
+- [nyxx_commands](https://pub.dev/packages/nyxx_commands): A command framework for handling both simple & complex commands.
+- [nyxx_pagination](https://pub.dev/packages/nyxx_pagination): Pagination support for nyxx.
+- [nyxx_lavalink](https://pub.dev/packages/nyxx_lavalink): Lavalink support for playing audio in voice channels.
+- [nyxx_extensions](https://pub.dev/packages/nyxx_extensions): Miscellaneous helpers for common situations when developing bots.
 
 ## More examples
 
-Nyxx examples can be found [here](https://github.com/nyxx-discord/nyxx/tree/dev/example).
-
-### Example bots
-- [Running on Dart](https://github.com/l7ssha/running_on_dart)
+- More examples can be found in our GitHub repository [here](https://github.com/nyxx-discord/nyxx/tree/main/example).
+- [Running on Dart](https://github.com/nyxx-discord/running_on_dart) is a complete example of a bot written in nyxx.
 
-## Documentation, help and examples
+## Additional documentation & help
 
-**Dartdoc documentation for latest stable version is hosted on [pub](https://www.dartdocs.org/documentation/nyxx/latest/)**
+The API documentation for the latest stable version can be found on [pub](https://pub.dev/documentation/nyxx).
 
-#### [Docs and wiki](https://nyxx.l7ssha.xyz)
-You can read docs and wiki articles for latest stable version on my website. This website also hosts docs for latest
-dev changes to framework (`dev` branch)
+### [Docs and wiki](https://nyxx.l7ssha.xyz)
+Tutorials and wiki articles are hosted here, as well as API documentation for development versions from GitHub.
 
-#### [Official nyxx discord server](https://discord.gg/nyxx)
-If you need assistance in developing bot using nyxx you can join official nyxx discord guild.
+### [Official nyxx Discord server](https://discord.gg/nyxx)
+Our Discord server is where you can get help for any nyxx packages, as well as release announcements and discussions about the library.
 
-#### [Discord API docs](https://discordapp.com/developers/docs/intro)
-Discord API documentation features rich descriptions about all topics that nyxx covers.
+### [Discord API docs](https://discord.dev/)
+Discord's API documentation details what nyxx implements & provides more detailed explanations of certain topics.
 
-#### [Discord API Guild](https://discord.gg/discord-api)
+### [Discord API Server](https://discord.gg/discord-api)
 The unofficial guild for Discord Bot developers. To get help with nyxx check `#dart_nyxx` channel.
 
 #### [Dartdocs](https://www.dartdocs.org/documentation/nyxx/latest/)
```

Comment:
```suggestion
#### [Pub.dev docs](https://pub.dev/documentation/nyxx)
```

Reply:
@abitofevrything If it's not intentional, can you also remove the last heading mark? Missed that